libfabric.vcxproj

```diff
@@ -474,4 +475,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
@hoopoepg @gladkovdmitry17 
I think editing the file in linux automatically removed the carriage return character (I guess). I'm not sure if there is way to prevent it. git diff shows the following:
```   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
```
Is this fine? The windows build passed.